infra/gcb/builds_status.py

```diff
@@ -183,37 +223,32 @@ def update_build_status(builds, projects, build_tag_suffix, status_filename):
   upload_status(successes, failures, status_filename)
 
 
-def update_build_badges(builds, projects, build_tag, coverage_tag):
-  for project in projects:
-    last_build = find_last_build(builds, project, build_tag)
-    last_coverage_build = find_last_build(builds, project, coverage_tag)
-    if not last_build or not last_coverage_build:
-      continue
-
-    badge = 'building'
-    if not is_build_successful(last_coverage_build):
-      badge = 'coverage_failing'
-    if not is_build_successful(last_build):
-      badge = 'failing'
+def update_build_badges(project, last_build, last_coverage_build):
```

Comment:
Update this so that it accepts `last_build_successful` and `last_coverage_build_successful` instead of expecting an entire build dict.